utils/build/docker/nodejs/express4-typescript/iast.ts

```diff
@@ -380,6 +381,145 @@ function initSourceRoutes (app: Express): void {
     readFileSync(vulnParam)
     res.send('OK')
   })
+
+  function getKafka () {
+    return new Kafka({
+      clientId: 'my-app-iast',
+      brokers: ['kafka:9092'],
+      retry: {
+        initialRetryTime: 100, // Time to wait in milliseconds before the first retry
+        retries: 20 // Number of retries before giving up
+      }
+    })
+  }
+
+  app.get('/iast/source/kafkavalue/test', (req: Request, res: Response): void => {
+    const kafka = getKafka()
+    const topic = 'dsm-system-tests-queue'
+    const timeout = 60000
+
+    let consumer: any
+    const doKafkaOperations = async () => {
+      consumer = kafka.consumer({ groupId: 'testgroup2' })
+
+      await consumer.connect()
+      await consumer.subscribe({ topic, fromBeginning: false })
+
+      const deferred: {
+        resolve?: Function,
+        reject?: Function
+      } = {
+        resolve: undefined,
+        reject: undefined
+      }
+
+      const promise = new Promise((resolve: Function, reject: Function): void => {
+        deferred.resolve = resolve
+        deferred.reject = reject
+      })
+
+      await consumer.run({
+        eachMessage: async ({ topic, partition, message }: { topic: string, partition: string, message: any }) => {
```

Comment:
`topic` and `partition` aren't used.

```suggestion
        eachMessage: async ({ message }: { message: any }) => {
```